BitwardenShared/UI/Auth/TwoFactorNotice/EmailAccess/EmailAccessView.swift

```diff
@@ -0,0 +1,100 @@
+import SwiftUI
+import SwiftUIIntrospect
+
+// MARK: - EmailAccessView
+
+/// A view that alerts the user to the new policy of sending emails to confirm new devices.
+///
+struct EmailAccessView: View {
+    // MARK: Properties
+
+    /// An environment variable for getting the vertical size class of the view.
+    @Environment(\.verticalSizeClass) var verticalSizeClass
+
+    /// The `Store` for this view.
+    @ObservedObject public var store: Store<EmailAccessState, EmailAccessAction, EmailAccessEffect>
+
+    var body: some View {
+        VStack(spacing: 12) {
+            DynamicImageTextStackView(minHeight: 0) {
+                Asset.Images.Illustrations.businessWarning.swiftUIImage
+                    .resizable()
+                    .frame(
+                        width: verticalSizeClass == .regular ? 152 : 124,
+                        height: verticalSizeClass == .regular ? 152 : 124
+                    )
+                    .accessibilityHidden(true)
+            } textContent: {
+                VStack(spacing: 16) {
+                    Text(Localizations.importantNotice)
+                        .styleGuide(.title, weight: .bold)
+
+                    Text(Localizations.bitwardenWillSendACodeToYourAccountEmail)
+                        .styleGuide(.title3)
+                }
+                .padding(.horizontal, 12)
+            }
```

Comment:
I'd looked around for some other view that did this and apparently didn't search hard enough 😅 yes, it can be replaced that way. Though now I'm curious if we could also migrate the [IntroCarouselView](https://github.com/bitwarden/ios/blob/main/BitwardenShared/UI/Auth/IntroCarousel/IntroCarouselView.swift#L58) to also use this, possibly with additional parameters in some way for the spacing?